webrtc.html

```diff
@@ -1771,7 +1846,7 @@ <h4>Set the RTCSessionDescription</h4>
                     </ol>
                   </li>
                   <li>
-                    <p>If <var>description</var> is set as a remote description,
+                    <p>Otherwise, if <var>remote</var> is <code>true</code>,
```

Comment:
The "otherwise" condition might also need to guard against description being null?